js/clinicalTimeline.js

```diff
@@ -169,15 +169,37 @@ var clinicalTimeline = (function(){
       .attr("width", overviewAxisWidth)
       .attr("class", "overview");
 
+    clinicalTimelineOverviewAxis(overviewSVG, getTickValues, minDays, maxDays, overviewAxisWidth, formatTime, daysToTimeObject);
+    clinicalTimelineVerticalLine(timelineBool.enableVerticalLine, beginning, ending, timelineBool.advancedView, timelineBool.tooltipOnVerticalLine);
+
     if(timelineBool.advancedView){
-      initAdvancedView(overviewSVG)
+      initAdvancedView(overviewSVG);
+      $("#advBtn").prop("disabled", true); //disable repeated clicking of advButton
+      $("#advBtn").addClass("active");
+      $("#simpleBtn").prop("disabled", false); //enable simpleBtn
+      $("#simpleBtn").removeClass("active");
+      $(".dropdown-toggle").prop("disabled", true);
+      $("#jsonInput").css("display", "block");
+      $("#tooltip-controller").css("visibility", "hidden");
+      $(".radio").css("visibility", "hidden");
+      if (location.href.indexOf("?view=simple") > -1) {  
+        window.history.replaceState('advancedView', '', location.href.replace("?view=simple", "?view=advanced"));
+      }
```

Comment:
I actually preferred having the buttons on `index.html`instead of in `clinicalTimeline.js`. That makes it more clear how `index.html` is using the `clinicalTImeline.js` API. If we were to include buttons for simple/advanced views or more complicated settings as part of the library I think the design should be a little more subtle than the dev style layout we have now. Also I'd rather see a separate function or file/plugin that adds these visual elements instead of adding a lot of code to `timeline()` in `clinicalTimeline.js`.